tapir/coop/templates/coop/log/update_share_owner_log_entry.html

```diff
@@ -1,4 +1,4 @@
-Update Share Owner:
```

Comment:
Maybe this should just say "member"? It's visible by members, they don't use "ShareOwner"